ksqldb-execution/src/test/java/io/confluent/ksql/execution/codegen/SqlToJavaVisitorTest.java

```diff
@@ -1046,8 +1046,8 @@ public void shouldGenerateCorrectCodeForNestedLambdas() {
             ArrayType.of(ParamTypes.DOUBLE),
             ParamTypes.DOUBLE,
             LambdaType.of(
-                ImmutableList.of(ParamTypes.DOUBLE, ParamTypes.DOUBLE),
-                ParamTypes.DOUBLE))
+                ImmutableList.of(ParamTypes.DOUBLE, ParamTypes.INTEGER),
```

Comment:
Not clear why we need such changes in this test?

Reply:
This test was written when we still had the restriction where we were getting the input type list from the ordering of the other function arguments. If you look at the FunctionCall arguments in the test, it's an Array<Double> and Integer so the previous InputTypeList was [Double, Integer]. To get that same input type list with the refactor, we need to update the LambdaType here.